FurEver/API Data/DogBreed.cs

```diff
@@ -17,14 +17,14 @@ public class DogBreedService
 
         public static async Task<List<DogBreed>> FetchBreedsAsync()
         {
-            var constants = Constant.Instace;
-            var dogBreedsAPI = constants.DogBReedsAPI;
+            var config = Constant.Instace;
+            var dogBreedsAPI = config.DogBReedsAPI;
```

Comment:
I'm indifferent on these two names.  They are both similar, and somewhat ambiguous.
Since the type (class) name is Constant, I'd be tempted to keep the variable named constant as well.
config could be anything, really.